packages/postcss-configs/README.md

```diff
@@ -1,4 +1,4 @@
-# @workleap/postcss-plugin
+# @workleap/postcss-configs
 
 All of Workleap’s default PostCSS plugins wrapped up in a single, easy-to-use plugin.
```

Comment:
Documentation is not accurate anymore, will write a new one when I come back from vacations.

Reply:
maybe we could log an issue to keep tracked of this

Reply:
Done!